src/main/java/com/openlattice/datastore/data/controllers/DataController.java

```diff
@@ -414,6 +417,51 @@ public Void clearEntitySet( UUID entitySetId ) {
         return null;
     }
 
+    @Override
+    @RequestMapping(
+            path = { "/" + ENTITY_SET + "/" + SET_ID_PATH + "/" + ENTITY_KEY_ID_PATH + "/graph" },
+            method = RequestMethod.DELETE
+    )
+    public Void clearEntityAndNeighborEntities(
+            @PathVariable( ENTITY_SET_ID ) UUID entitySetId,
+            @PathVariable( ENTITY_ID ) UUID entityId
```

Comment:
should be `entityKeyId` to avoid confusion with entity ids (the ones for integrations)

Reply:
(same for `neighborEntityId` and `associationEntityId`)

Reply:
What's the difference between `entityId` and `entityKeyId`? I thought `entityId` referred to the id of the actual piece of data, the entity?